interfaces/Portalicious/src/app/pages/project/project-team/add-user-button/add-user-button.component.html

```diff
@@ -1,7 +1,94 @@
 <p-button
-  label="Add user to team"
-  i18n-label
-  rounded
+  label="Add team member"
+  i18n-label="@@add-team-member"
+  outlined
   icon="pi pi-plus"
-  (click)="onAdd()"
+  class="text-xs sm:text-base"
```

Comment:
I don't think we need to cater to small screens to this extent. If we must, then I would rather do it globally for primeng components in `styles.scss`. But again, I'd rather we didn't at all, and instead we tackle this systematically if and when it truly becomes relevant.

Reply:
See comment above @arsforza ...